demo.html

```diff
@@ -1,53 +1,64 @@
 <!doctype html>
 <html>
+
 <head>
-  <title>swipeable-card</title>
+    <title>swipeable-card</title>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+    <meta name="mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-capable" content="yes">
 
-  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
-  <meta name="mobile-web-app-capable" content="yes">
-  <meta name="apple-mobile-web-app-capable" content="yes">
+    <script src="../webcomponentsjs/webcomponents.js"></script>
 
-  <script src="../webcomponentsjs/webcomponents.js"></script>
-  
-  <link rel="import" href="swipeable-card.html">
+    <link rel="import" href="swipeable-card.html">
 
-  <style>
-  
+    <style>
     #cards {
-      margin: 0 auto;
-      background: #eee;
-      overflow: auto;
-      -webkit-user-select: none;
-      -moz-user-select: none;
+        margin: 0 auto;
+        background: #eee;
+        overflow: auto;
+        -webkit-user-select: none;
+        -moz-user-select: none;
     }
-    
     swipeable-card {
-      height: 200px;
-      width: 240px;
-      margin: 20px auto;
-      padding: 30px;
+        height: 200px;
+        width: 240px;
+        margin: 20px auto;
+        padding: 30px;
     }
-      
-  </style>
+    </style>
 
 </head>
+
 <body unresolved>
 
-  <div id="cards">
-  
-    <swipeable-card>Swipe Me!</swipeable-card>
-    <swipeable-card>Swipe Me!</swipeable-card>
-    <swipeable-card disableSwipe></swipeable-card>
-    <swipeable-card>Swipe Me!</swipeable-card>
-    <swipeable-card disableSwipe></swipeable-card>
-    
-  </div>
-  
-  <script>
+    <div id="cards">
+
+        <swipeable-card>Swipe Me and I disappear!</swipeable-card>
+        <swipeable-card id="renewable">Swipe Me and I'll go to the end of the deck!</swipeable-card>
+        <swipeable-card disableSwipe>I don't swipe!</swipeable-card>
+        <swipeable-card id="renewable">Swipe Me and I'll go to the end of the deck!</swipeable-card>
+        <swipeable-card disableSwipe>I don't swipe!</swipeable-card>
+
+    </div>
+
+    <script>
     document.querySelector('#cards').addEventListener('swipeable-card-swipe-away', function(e) {
-      e.target.parentNode.removeChild(e.target);
+        e.target.parentNode.removeChild(e.target); // removes this card
+        if (e.target.id == "renewable") {
```

Comment:
It might cool to express this behavior in the form of a Polymer element, rather than just a chunk of script. For example, a `renewable-card-list` or something similar.